### PR TITLE
Jdh gameplay

### PR DESCRIPTION
**Pull request by CygnusRoboticus**

### CODE REVIEW

File: app/lib/engine/plugins/editor-menu-plugin.ts [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+  public state!: StatePlugin;
+
+  public toggle!: Phaser.GameObjects.Text;
+  public mainMenu!: Phaser.GameObjects.Text;
+
+  constructor(scene: Phaser.Scene, pluginManager: Phaser.Plugins.PluginManager) {
+    super(scene, pluginManager);
+  }
+
+  public boot() {
```
Comment (cwhittemore1) [inline app/lib/engine/plugins/editor-menu-plugin.ts]: Can we make every function in this file global?  🌎

Comment (CygnusRoboticus) [reply app/lib/engine/plugins/editor-menu-plugin.ts]: No

Comment (cwhittemore1) [reply app/lib/engine/plugins/editor-menu-plugin.ts]: 🆗

Comment (cwhittemore1) [reply app/lib/engine/plugins/editor-menu-plugin.ts]: 🐸 😞

**Pull request merged**